crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -73,15 +73,15 @@ def version
       def addons
         [].tap do |list|
           ["ceph", "ha"].each do |addon|
-            list.push(addon) if addon_installed?(addon) && addon_enabled?(addon)
+            list.push(addon) if addon_installed?(addon) && addon_deployed?(addon)
           end
         end
       end
 
       def ceph_status
         {}.tap do |ret|
           ceph_node = ::Node.find("roles:ceph-mon AND ceph_config_environment:*").first
-          return ret if ceph_node.nil?
+          next unless ceph_node
```

Comment:
yes, it is. it's just exiting the block. we have it in other places as well, that's why i changed it